ui/src/app/dashboard/grouped-summary/grouped-summary.component.html

```diff
@@ -0,0 +1,24 @@
+<mat-card *ngFor="let aggregation of aggregationResponse.aggregations">
```

Comment:
Actually, this is not what I was expecting. I was expecting that `GroupedSummaryComponent` would == 1 card, then you would render several of them from the dashboard component.